### PR TITLE
Correct SERREAS derivation in ael03

### LINKED ISSUE #269: SERREAS derivation in AEL03 needs correction
There can be multiple reasons why an AE was serious, e.g. required hospitalization and was life-threatening. In these cases the listing should display both these reasons as "2, 3", but with the current case-when setup it would only ever display one reason.

### PR DESCRIPTION
**Pull request by Melkiades**
Fixes #269

### CODE REVIEW

File: book/listings/adverse-events/ael03.qmd [hunk 1]
```diff
@@ -106,6 +113,7 @@ lsting <- as_listing(
 head(lsting, 20)
 ```
 
+
```
Comment (pawelru) [inline book/listings/adverse-events/ael03.qmd:116]:
```suggestion
```

File: book/listings/adverse-events/ael03.qmd [hunk 2]
```diff
@@ -12,6 +12,7 @@ library(dplyr)
 library(rlistings)
 
 adae <- random.cdisc.data::cadae
+adae$AESCONG[which(adae$AESER == "Y")[seq(3)]] <- "Y" # To mimick correct possibility of overlap
```
Comment (edelarua) [inline book/listings/adverse-events/ael03.qmd:15]: Spelling
```suggestion
adae$AESCONG[which(adae$AESER == "Y")[seq(3)]] <- "Y" # To mimic possibility of overlap
```

File: book/listings/adverse-events/ael03.qmd [hunk 3]
```diff
@@ -38,17 +39,23 @@ out <- adae %>%
...
+      which(
+        c(AESDTH, AESLIFE, AESHOSP, AESDISAB, AESCONG, AESMIE) == "Y"
+      ),
+      collapse = ", "
+    )
+  ) %>%
+  ungroup()
+
+# Check correct derivation of SERREAS
+# out |> select(AESDTH, AESLIFE, AESHOSP, AESDISAB, AESCONG, AESMIE, SERREAS)
```
Comment (edelarua) [inline book/listings/adverse-events/ael03.qmd:56]: Remove commented code
```suggestion
```

File: book/listings/adverse-events/ael03.qmd [hunk 4]
```diff
@@ -118,6 +125,7 @@ webr_code_labels <- c("lsting")
 ```{r setup}
 #| code-fold: show
 ```
+
```
Comment (edelarua) [inline book/listings/adverse-events/ael03.qmd:128]: Empty line
```suggestion
```

**Review by edelarua — CHANGES_REQUESTED** [review]
Looks good! Just a few small suggestions, and can you please update the snapshot as well? Then it should be good to go!

**Comment by shajoezhu** [comment]
hi @Melkiades , can you update the snapshot please. Thank you!

**Review by pawelru — APPROVED** [review]

**Pull request merged**